keystone.go

```diff
@@ -51,23 +91,101 @@ func NewKeystoneClient(auth_url, tenant_name, username, password, token string)
 		username,
 		password,
 		token,
-		nil,
+		"",
+		"",
+		"",
+		false,
 	}
 }
 
 func NewKeepaliveKeystoneClient(auth_url, tenant_name, username, password, token string) *KeepaliveKeystoneClient {
-	return &KeepaliveKeystoneClient {
+	return &KeepaliveKeystoneClient{
 		KeystoneClient{
 			auth_url,
 			tenant_name,
 			username,
 			password,
 			token,
-			nil,
```

Comment:
I suggest  using named fields in initialisation then adding empty values

Reply:
I was planning on not to change the existing prototypes.That's why added empty values.